client/components/gravatar-with-hovercards/hovercard-content.jsx

```diff
@@ -106,18 +140,69 @@ function HovercardContent( props ) {
 										className="gravatar-hovercard__primary-blog-card-follow-button"
 										siteUrl={ primaryBlogUrl }
 										hasButtonStyle
-										followSource="gravatar-hovercard"
+										followSource="gravatar-hovercard__primary-blog-card"
 									/>
 								</div>
 							) }
 						</div>
 
 						<div className="gravatar-hovercard__footer">
-							{ /* TODO: Add recommended blogs list */ }
+							{ shouldShowRecommendedBlogs && (
+								<div className="gravatar-hovercard__recommended-blogs">
+									<div className="gravatar-hovercard__recommended-blogs-header">
+										<h5 className="gravatar-hovercard__recommended-blogs-title">
+											{ translate( 'Recommended blogs' ) }
+										</h5>
+										<a
+											className="gravatar-hovercard__recommended-blogs-view-all"
+											href={ recommendedBlogsLink }
+											onClick={ ( e ) => {
+												e.preventDefault();
+												page( recommendedBlogsLink );
+											} }
+										>
+											{ translate( 'View all' ) }
+										</a>
+									</div>
+									<ul className="gravatar-hovercard__recommended-blogs-list">
+										{ shuffle( recommendedBlogs )
```

Comment:
Can we memoize the shuffle but still get random results on the next hover? 🤔 

```
const shuffledBlogs = useMemo( () => {
    if ( ! recommendedBlogs?.length ) return [];
    return shuffle( recommendedBlogs ).slice( 0, 3 );
}, [ recommendedBlogs ] );
```

Reply:
Got it! Yeah that wouldn't help on remount. And lots of re-rendering for the brief time the hover card is visible doesn't seem likely.